### DISCUSSION TITLE
Change text alignment on dark theme

### DISCUSSION THREAD

**Discussion by vesuvi88** [tneotia/html-editor-enhanced#262] [Q&A]
https://user-images.githubusercontent.com/19634035/170237314-933322fd-9327-49ca-b2fa-e66fc45cc16d.mp4

Hi all!
I am faced with the following problem: when I change the alignment of the text, its background turns white. System brightness: dark.

In html document i have this:
`<div style="text-align: center;"><span style="color: rgb(0, 0, 0); background-color: rgb(255, 255, 255);">Dhgfdhgfdhgfd</span></div>`

This is a bug, or my mistake of setting plugin?

**Comment by tneotia** [comment]
I've noticed this as well, but haven't been able to work out a fix. The white background color seems to be automatically added, for whatever reason.
